server/@types/dto/index.d.ts

```diff
@@ -61,7 +64,9 @@ declare module 'dto' {
     prisonNumber: string
     prisonId: string
     strengthTypeCode: string
-    detail?: string
+    symptoms?: string
```

Comment:
Change to the previously committed StrengthDto .... not sure where I got "detail" from, but in a very similar way to Challenges, the DTO needs to encapsulate symptoms, how it was identified, and an "other" field.